tests/domain/usecases/fitbit/test_usecase_process_daily_activities.py

```diff
@@ -98,7 +98,7 @@ class DailyActivityScenario:
           fields:
             - distance
           daily_goals:
-            distance_km: 100
+            distance_km: 18
```

Comment:
ℹ️ In the test data:
* the all-time high is 20km.
* "today" is 15km

To reproduce the bug, we need a value between the two, for our goal:
* higher than today's value, which means we didn't meet the goal.
* lower than any other previous value, which means at some previous point we did meet the goal.